pontoon/base/views.py

```diff
@@ -988,50 +993,75 @@ def transvision(request):
         return HttpResponse("error")
 
 
+def get_download_content(slug, code, relative_path):
```

Comment:
Translation table is updated often - every time a translation is added, deleted, approved or unfuzzied. Either by a user or by the sync job.